packages/integrations/mdx/README.md

```diff
@@ -222,6 +223,46 @@ const { frontmatter } = Astro.props;
 </html>
 ```
 
+You can also add type safety using [the `Props` type](/en/guides/typescript/#component-props) with the `MDXLayoutProps` helper.
+
+_**Note:** `MDXLayoutProps` is the same as the `MarkdownLayoutProps` utility type, with `rawContent()` and `compiledContent()` removed (since these are not available for `.mdx` files). If you understand this difference, feel free to use `MarkdownLayoutProps` instead when sharing a layout across `.md` and `.mdx` files._
+
+```astro ins={2,4-9}
+---
+// src/layouts/BaseLayout.astro
+import type { MDXLayoutProps } from 'astro';
+
+type Props = MDXLayoutProps<{
```

Comment:
Are we meant to export/use `Props` here?

Reply:
No need to `export` these days! Have a task to remove `export` from our docs across the board.